src/workerd/server/workerd-api.h

```diff
@@ -38,11 +39,14 @@ class WorkerdApi final: public Worker::Api {
       kj::Own<JsgIsolateObserver> observer,
       api::MemoryCacheProvider& memoryCacheProvider,
       const PythonConfig& pythonConfig,
-      kj::Maybe<kj::Own<jsg::modules::ModuleRegistry>> newModuleRegistry);
+      kj::Maybe<kj::Own<jsg::modules::ModuleRegistry>> newModuleRegistry,
+      kj::Maybe<kj::Own<VirtualFileSystem>> vfs);
```

Comment:
Is this really a Maybe? Seems like we'll have it always so it should probably just be unwrapped.

Reply:
PS: if this is just for tests then I think we can just add a fixture that's a no-op VFS instead.
Our code should signify what is possible in prod rather than what is useful in tests (in my opinion).

Reply:
Currently it's a maybe because nothing on the internal repo side sets this. Once the internal project is updated to provide the vfs instance this can be changed removing the kj::Maybe.

Reply:
Is this expected to be part of this PR before pulling out of draft?
If so, mind adding it as a TODO tag to the PR description?